app/controllers/albums_controller.rb

```diff
@@ -66,4 +68,8 @@ def set_album
     def album_params
```

Comment:
I think the params function should only permit attributes that the model(or the table representing it), holds directly, not throught association.
So images array should not be permitted.

Reply:
The official guide uses it in the params: [link](https://edgeguides.rubyonrails.org/active_storage_overview.html#has-many-attached), and in my experience creating an album without it fails, as there are no images getting uploaded.

Reply:
The example in the guide for the general use. It is a viable solution if we alaways create / update with the same parameters. But we implemented a custom logic for handling optional images uplodads which makes this a different use-case.
We don't use the `params[:alubum][:images]` as a regular parameter so it should be separate I think. 

We should be able to create albums without images because images is not a required attribute for album.
I tried it and it is posible to create an album without any image:
![Screenshot from 2021-02-09 20-23-19](https://user-images.githubusercontent.com/48379676/107416577-ed51a180-6b14-11eb-886d-73070dcbeab7.png)

Reply:
Oh, i see. My problem was that if I left it out I wasn't able to create an album with images, only an empty one, and then i could use the update endpoint to upload images.